cmd/juju/commands/run_test.go

```diff
@@ -151,6 +171,10 @@ func (*RunSuite) TestTimeoutArgParsing(c *gc.C) {
 		message: "two hours",
 		args:    []string{"--timeout=2h", "--all", "sudo reboot"},
 		timeout: 2 * time.Hour,
+	}, {
+		message: "5 minutes",
+		args:    []string{"-t","5m", "--all", "sudo reboot"},
```

Comment:
Looks like a few whitespace issues, did you run go fmt ./... before pushing?
Ideally you'd have the git precommit hook set up to do this and also run go vet